pages/developers/pytest.md

```diff
@@ -23,7 +23,7 @@ Python used to have three major choices for tests; but now [pytest][] is used al
 
 > ### What about other choices?
 >
-> The alternative library, "nose", has been abandoned in favor of pytest, which can run nose-style tests. The standard library has a test suite as well, but it's extremely verbose and complex; and since "developers" run tests, your test requirements don't affect users. And pytest can run stdlib style testing too. So just use pytest. All major packages use it too, including NumPy. Most other choices, like [Hypothesis][], are related to pytest.
+> The alternative library, "nose", has been abandoned in favor of pytest, which can run nose-style tests. The standard library has a test suite as well, but it's extremely verbose and complex; and since "developers" run tests, your test requirements don't affect users. And pytest can run stdlib style testing too. So just use pytest. All major packages use it too, including NumPy. Most other choices, like [Hypothesis][], are related to pytest and just extend it
```

Comment:
```suggestion
> The alternative library, `nose`, has been abandoned in favor of `pytest`, which can run nose-style tests. The standard library has a test suite as well, but it's extremely verbose and complex; and since "developers" run tests, your test requirements don't affect users. And `pytest` can run stdlib style testing too. So just use `pytest`. All major packages use it too, including `NumPy`. Most other choices, like [Hypothesis][], are related to `pytest` and just extend it.
```